src/YamahaAPI.ts

```diff
@@ -67,12 +67,12 @@ interface Response {
 export interface StatusResponse {
     response_code: number;
     power: string;
-    //sleep: number;
+    sleep: number;
```

Comment:
probably that was not intended to be in this PR?
```suggestion
    //sleep: number;
```

Reply:
Just though it can be maybe useful if I want to implement some additional features )